tests/store/handlers/test_readhandler.py

```diff
@@ -297,30 +297,35 @@ def test_archives_not_returned_via_retrieval_id(retrieval_task_id: int, populate
 
 
 @pytest.mark.parametrize(
-    "retrieved_at, should_be_returned",
+    "retrieved_at, tags, should_be_returned",
     [
-        (datetime.datetime(year=2023, month=12, day=12), False),
-        (datetime.datetime(year=2023, month=1, day=1), True),
-        (None, False),
+        (datetime.datetime(year=2023, month=12, day=12), [], False),
+        (datetime.datetime(year=2023, month=1, day=1), [], True),
+        (datetime.datetime(year=2023, month=1, day=1), ["spring"], True),
+        (datetime.datetime(year=2023, month=1, day=1), ["fastq"], False),
+        (None, [], False),
     ],
```

Comment:
If you want you can add `ids` with a short description to the different parameter sets so that they are identified better, this could be a good case to use that

Reply:
Good suggestion, added!